agent-install/agent-install.sh

```diff
@@ -1301,7 +1306,10 @@ function get_all_variables() {
             lastpart=$(echo $IMAGE_ON_EDGE_CLUSTER_REGISTRY | cut -d "/" -f 3) # <arch>_anax_k8s
             image_arch_in_param=$(echo $lastpart | cut -d "_" -f 1)
             if [[ "$image_arch" != "$image_arch_in_param" ]]; then
-                log_fatal 1 "Cannot use agent image with $image_arch_in_param arch to install on $image_arch cluster, please use agent image with '$image_arch'"
+                log_warning "Cannot use agent image with $image_arch_in_param arch to install on $image_arch cluster, will change arch to '$image_arch'"
+                part1=$(echo $IMAGE_ON_EDGE_CLUSTER_REGISTRY | cut -d "/" -f 1)
+                part2=$(echo $IMAGE_ON_EDGE_CLUSTER_REGISTRY | cut -d "/" -f 2)
+                IMAGE_ON_EDGE_CLUSTER_REGISTRY=$part1/$part2/${image_arch}_anax_k8s
```

Comment:
I think it would be better if we don't assume that IMAGE_ON_EDGE_CLUSTER_REGISTRY always has 3 parts. I used docker.io/dlarson04/horizon/amd64_anax_k8s one time and this failed. Can we add more code to find which part has *_anax_k8s ?